src/Core/Grid/Definition/Factory/OrderGridDefinitionFactory.php

```diff
@@ -236,6 +239,7 @@ protected function getColumns()
                 ->setOptions([
                     'field' => 'date_add',
                     'format' => $this->contextDateFormat,
+                    'clickable' => true,
```

Comment:
Already clickable

Reply:
Euuuuh no, `DateTimeColumn` default value for `clickable` is false

Reply:
See https://github.com/PrestaShop/PrestaShop/pull/16318/files#diff-4ecc9ae27c544deb5b13a5da1d8eec2fR56